src/extension/views/json/json.js

```diff
@@ -220,6 +220,32 @@ export class JSONView extends LitElement {
     return elements;
   }
 
+  /**
+   * Finds the row with the most keys and returns key names as headers.
+   * @param {Object[]} rows The rows
+   * @returns {string[]} The header names
+   */
+  getHeaders(rows) {
+    const sortedRows = rows.sort((a, b) => Object.keys(b).length - Object.keys(a).length);
+    return Object.keys(sortedRows[0]);
```

Comment:
I think it would be better to iterate over all rows, and build the list of all possible keys. probably similarly expensive than sorting with this expensive comparator ;-)

Reply:
Now using `columns` from payload instead